<launch>

    <arg name="joy_dev" default="/dev/input/js0"/>

    <!-- FCU url to local simulation-->
    <arg name="fcu_url" default="udp://:14540@127.0.0.1:14557" />

    <!-- MAVROS-->
    <include file="$(find px4)/launch/mavros_posix_sitl.launch">
        <arg name="fcu_url" value="$(arg fcu_url)" />
    </include>

    <!-- Joystick node-->
    <node pkg="joy" type="joy_node" name="joy_node"  required="true" output="screen">
            <param name="dev" type="string" value="$(arg joy_dev)" />
    </node>

    <!-- Your package node-->
    <!-- <node pkg="mypackage" type="setpoints_node.py" name="setpoints_node"  required="true" output="screen">
    </node> -->

    <!-- Velocity controller node-->
    <node pkg="drone_laser_alignment" type="velocity_controller.py" name="velocity_controller"  required="true" output="screen">
    </node>

    <!-- Laser alignment node-->
    <node pkg="drone_laser_alignment" type="laser_alignment.py" name="laser_alignment"  required="true" output="screen">
    </node>

</launch>